src/User.php

```diff
@@ -130,6 +130,20 @@ public static function getAdditionalMenuOptions()
         return false;
     }
 
+    public static function getAdditionalMenuLinks()
+    {
+        $links = [];
+        if (Auth::useAuthExt() && Session::haveRight('user', self::IMPORTEXTAUTHUSERS)) {
+            if (static::canCreate()) {
+                $ext_auth_label = __s('Add from an external source');
+                $links['<i class="ti ti-user-cog"></i>' . $ext_auth_label] = 'user.form.php?new=1&amp;ext_auth=1';
+            }
+            if (static::canCreate() || static::canUpdate()) {
+                $links['<i class="ti ti-settings me-2"></i>' . __s('LDAP directory link')] = "ldap.php";
```

Comment:
```suggestion
                $links['<i class="ti ti-user-cog"></i><span>' . $ext_auth_label . '</span>'] = 'user.form.php?new=1&amp;ext_auth=1';
            }
            if (static::canCreate() || static::canUpdate()) {
                $links['<i class="ti ti-settings"></i><span>' . __s('LDAP directory link') . '</span>'] = "ldap.php";
```

this to make auto spacing with the icon.
I didn't make all other iterations, but this need to be applied on all getAdditionalMenuLinks